Spokestack/TFLiteKeywordRecognizer.swift

```diff
@@ -72,7 +72,16 @@ import TensorFlowLite
         self.encodeState.fill(0.0) // fill now because the encoded state is used to feed-forward
         let encodeLength = c.keywordEncodeLength * c.sampleRate / 1000 / self.hopLength
         self.encodeWindow = RingBuffer(encodeLength * c.keywordEncodeWidth, repeating: -1.0)
-        self.classes = self.configuration.keywords.components(separatedBy: ",")
+        do {
+            let keywords = try CommandModelMeta(configuration)
+            self.classes = keywords.model.classes.flatMap({ c in
```

Comment:
We don't want the utterance text here, just the class names—the model outputs will correspond to training classes, not individual utterances that resolve to those classes.

Reply:
🤦  good catch, thanks